pxemgr/pxemanager.go

```diff
@@ -212,6 +214,9 @@ func (mgr *pxeManagerT) startIPXEserver() error {
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/set_etcd_cluster_token").HandlerFunc(withSerialParam(mgr.setEtcdClusterToken))
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/override").HandlerFunc(withSerialParam(mgr.override))
 
+	mgr.router.Methods("GET").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.getMayuConfig)
+	mgr.router.Methods("PUT").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.setMayuConfig)
```

Comment:
I won't keep you from applying rules/guidelines from the public API to internal APIs. However I don't think we have to be as strict and as forward-thinking about the design and consistency. And also, I won't act as the rule keeper for our internal projects. I happily do this for the public API as that is customer-facing.